README.md

```diff
@@ -1,2 +1,41 @@
-# radiometric_normalization
-Implementation of radiometric normalization workflows
+# Radiometric Normalization #
+
+This library contains the functions to analyse a set of reference images for features that have a stable level of intensity over the whole reference set and use these features to normalize the intensity distribution of separate candidate image.
+
+It was originally created to radiometrically normalize a satellite image to a time series from a reference
+(atmospherically corrected) dataset.
+
+## How to use ##
+
+This git repo contains a vagrant environment within which to run the library in. To start the vagrant environment, in the root directory of the repo type:
```

Comment:
"a vagrant environment to run the library in" or "a vagrant environment within which to run the library"